docs/examples/prototypes.md

```diff
@@ -1,5 +1,57 @@
+| Hotel Booking               | Hotel Search, Hotel Booking                                                                                | Roby on Rails        | [amadeus-hotel-booking-rubyonrails](https://github.com/amadeus4dev/amadeus-hotel-booking-rubyonrails)                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| Hotel Search                | Hotel Search, Points of Interest, Safe Place                                                               | Python, django       | [amadeus-hotel-area-safety-pois-django](https://github.com/amadeus4dev/amadeus-hotel-area-safety-pois-django)                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| Hotel Search                | Hotel Search                                                                                               | Swift                | [amadeus-hotel-search-swift]("https://github.com/amadeus4dev/amadeus-hotel-search-swift")                                       |
+| Itinarary Management        | Trip Purpose Prediction                                                                                    | Python, django       | [amadeus-trip-purpose-django](https://github.com/amadeus4dev/amadeus-trip-purpose-django)                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| On demand flight status     | On-Demand Flight Status                                                                                    | Python               | [amadeus-async-flight-status]("https://github.com/amadeus4dev/amadeus-async-flight-status")                                |
+
+
+
+
+### Prototypes from community
+
+Also, we have much more prototypes that developers in our community built and shared! 
+
+Explore more in [Amadeus for Developers Github](https://github.com/amadeus4dev) by searching the keyword 'prototype', 'demo' or 'example'. 
```

Comment:
GitHub here as well. 
I would suggest to use `prototype` , `demo`, `example` markdown way of quoting